### PR TITLE
feat - landing page

### PR DESCRIPTION
**Pull request by Ayesha-7**

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]

File: Styles/notes.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/* home.css
+@media screen and (max-width: 768px) {
```
Comment (SukanyaSen530) [inline Styles/notes.css:2]: You can remove it if not required.

**Review by SukanyaSen530 — APPROVED** [review]

File: Styles/header.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  color: var(--text-color);
+  padding: 0;
+}
+
+.nav-logo {
+  margin: 1rem;
+}
+
+.nav-search {
+  min-width: 30%;
```
Comment (SukanyaSen530) [inline Styles/header.css:12]: I think it should be atleast 80-90% as the X symbol will show half way in the UI

**Review by tanishk15sharma — APPROVED** [review]
You can remove comments and suggestions given above , rest looks good to me .

**Review by SanjayJatti — APPROVED** [review]
- Code is clean
- You should use component library classes as much as possible and try to optimize CSS
- UX is looking amazing

File: Styles/notes.css (hunk 1 above)
Comment (Ayesha-7) [reply Styles/notes.css:2]: yes, will remove this.

File: Styles/header.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  padding: 0.5rem;
+}
+
+.search-container > * {
+  background-color: transparent;
+  border: none;
+}
+
+.search-container input {
+  outline: transparent;
```
Comment (SukanyaSen530) [inline Styles/header.css:32]: It's  width can be increased so it takes up full width. :)

Comment (Ayesha-7) [reply Styles/header.css:32]: thanks for pointing out, made the suggested change.

**Pull request merged**